src/service/userService.ts

```diff
@@ -25,7 +24,7 @@ class UserService {
         this.http = http;
     };
 
-    async signup(userData: UserSignupBody): Promise<any>  {
+    async signup(userData: UserSignupBody): Promise<LoginResponse>  {
```

Comment:
The idea of typescript is to explicitly state the type of the data. It's best to be verbose. So don't use LoginResponse on a SignUp Response. Create a new type even if it is the same typing. Things can change in the future.

Reply:
Fixed this, are we good?